scripts/utils/formatNumber.js

```diff
@@ -0,0 +1,19 @@
+export default (number, isPercentage)  => {
+  let parsedNumber;
+
+  if (number === null || number === 'undefined' || typeof number !== 'number') return 'N/A';
+
+  if (number === 0) return number;
+
+  if (isPercentage) {
+    parsedNumber = (number * 100).toFixed(1);
+  } else {
+    if (number > 0 && number < 1) {
+      parsedNumber = number.toFixed(4);
+    } else {
+      parsedNumber = number.toFixed(2);
```

Comment:
On the sankey we are using 1 decimal, not 2. Are you sure the clients wants 2? If so we need to change it in `niceNumber` as well